File: java-samples/corda5-negotiation-cordapp/workflows/src/main/java/com/r3/developers/samples/negotiation/workflows/modify/ModifyFlowArgs.java
package com.r3.developers.samples.negotiation.workflows.modify;

import net.corda.v5.base.annotations.CordaSerializable;

import java.util.UUID;

@CordaSerializable
public class ModifyFlowArgs {
    private UUID proposalID;
    private int newAmount;

    public ModifyFlowArgs() {
    }

    public ModifyFlowArgs(UUID proposalId, int newAmount) {
        this.proposalID = proposalId;
        this.newAmount = newAmount;
    }

    public int getNewAmount() {
        return newAmount;
    }

    public UUID getProposalID() {
        return proposalID;
    }


}

File: kotlin-samples/corda5-negotiation/workflows/src/main/kotlin/com/r3/developers/samples/negotiation/workflows/Modify/ModifyFlowArgs.kt
package com.r3.developers.samples.negotiation.workflows.Modify

import net.corda.v5.base.annotations.CordaSerializable
import java.util.*

@CordaSerializable
data class ModifyFlowArgs (
    var proposalID: UUID,
    var newAmount: Int
)
